lib/amazon.js

```diff
@@ -24,7 +24,47 @@ var ERRORS = {
 	}
 };
 
-var VALIDATION_PATH = 'https://appstore-sdk.amazon.com/version/' + VER + '/verify/developer/' + SECRET + '/user/' + UID + '/purchaseToken/' + PTOKEN;
+var VALIDATION_PATH = 'https://appstore-sdk.amazon.com/version/' + VER + '/verifyReceiptId/developer/' + SECRET + '/user/' + UID + '/receiptId/' + PTOKEN;
 var RENEW_PATH = 'https://appstore-sdk.amazon.com/version/' + VER + '/renew/developer/' + SECRET + '/user/' + UID + '/purchaseToken/' + PTOKEN;
 
 
+module.exports.validatePurchase = function (receipt, cb) {
+	VALIDATION_PATH = VALIDATION_PATH.replace('{purchaseToken}', receipt); // IF IT DIDN'T PASS AS CONFIG VALUE, REPLACE IT HERE
```

Comment:
Shouldn't this be receiptId instead of receipt?
It is kinda confusing since we have receipts already